core/dbt/contracts/graph/nodes.py

```diff
@@ -615,6 +619,7 @@ class ModelNode(CompiledNode):
     constraints: List[ModelLevelConstraint] = field(default_factory=list)
     version: Optional[NodeVersion] = None
     latest_version: Optional[NodeVersion] = None
+    state_relation: Optional[RelationalNode] = None
```

Comment:
As mentioned in the PR description, `state_relation` is a very general name that doesn't explain much of what this field holds. Please feel free to suggest better names for this attribute.

Reply:
Maybe `other_relation`, `target_relation`, `cross_schema_relation`

Reply:
Spoke with Gerda about this name and she thought it made sense for the context it was in.